.github/workflows/code_freeze.yml

```diff
@@ -9,4 +9,4 @@ jobs:
     steps:
     - name: Fail if Code Freeze is enabled
       run: |
-        exit 0
+        exit 1
```

Comment:
This should be reverted back to `exit 0`

Reply:
Sounds good!